app/dashboard/templates/shared/bounty_nav.html

```diff
@@ -16,39 +16,39 @@
 
 {% endcomment %}
 
-                            <div class="w-100" style="text-align: center;">
-                                <ol class="bounty_nav">
-                                    <li class="{% if active_item == 1 %}active{%endif%} submit" >
-                                        1.
-                                        {% if past_tense %}
-                                            Open
-                                        {%else%}
-                                            Create Work
-                                        {%endif%}
-                                    </li>
-                                    <li class="{% if active_item == 2 %}active{%endif%} fulfill">
-                                        2.
-                                        {% if past_tense %}
-                                            Work Started
-                                        {%else%}
-                                            Start Work
-                                        {%endif%}
-                                    </li>
-                                    <li class="{% if active_item == 3 %}active{%endif%} fulfill">
-                                        3.
-                                        {% if past_tense %}
-                                            Work Submitted
-                                        {%else%}
-                                            Submit Work
-                                        {%endif%}
-                                    </li>
-                                    <li class="{% if active_item == 4 %}active{%endif%} accept">
-                                        4.
-                                        {% if past_tense %}
-                                            Work Done
-                                        {%else%}
-                                            Accept Work
-                                        {%endif%}
-                                    </li>
-                                </ol>
-                            </div>
+<div class="w-100" style="text-align: center;">
+    <ol class="bounty_nav">
```

Comment:
Can you switch this to 2-space indentation?

Reply:
it looks like the editorconfig prescribes css, html, scss, and py to be 4 space?
```
# 4 space indentation
[*.{css,html,scss,py}]
indent_size = 4
```

Reply:
Whoops. Thought we updated that.  I'll push an update for it.  If you can get to the indentation change for that file (I only ask because the lines are changing anyway), feel free. If not, I can change it following the editorconfig changes after this is merged.

Reply:
Sure, I can do it -- just wanted to make sure we were on the same page re: config